tensorflow_datasets/scripts/document_datasets_test.py

```diff
@@ -40,20 +40,20 @@ class DummyMnistConfigs(DummyMnist):
 
 
 class DocumentDatasetsTest(testing.TestCase):
-
+  """Test for document_datasets module"""
   @classmethod
-  def setUpClass(cls):
+  def setUpClass(cls): # pylint: disable = invalid-name
```

Comment:
It is in camel case. Please change this don't disable it.

Reply:
Thanks for the review but I feel that we should not change the naming format here because it is a common practice to name the class setup methods in such way